creating/creating_index.adoc

```diff
@@ -671,26 +671,22 @@ include::help.adoc[]
 
 === Creating a Changelog
 
-=== Verifying Dockerfiles (linter)
+=== The Dockerfile linter
 
-The Dockerfile-lint is a rule based 'linter' for verifying Dockerfiles. The rules are used to check syntax and best practice options for things such as:
+==== What is the linter?
 
-.Sample Rules
+The Dockerfile-lint is a rule based link:https://en.wikipedia.org/wiki/Lint_(software)['linter'] for verifying Dockerfiles. The rules are used to check file syntax and best practice options for things such as:
+
+.Example Sample Rules
```

Comment:
Not 100% sure how this gets rendered, but it might not be readable very well with the title.